programs/test/udp_proxy.c

```diff
@@ -555,6 +653,36 @@ int handle_message( const char *way,
 
     id = cur.len % sizeof( dropped );
 
+    if( strcmp( way, "S <- C" ) == 0 )
+    {
+        delay_list     = opt.delay_cli;
+        delay_list_len = opt.delay_cli_cnt;
+    }
+    else
+    {
+        delay_list     = opt.delay_srv;
+        delay_list_len = opt.delay_srv_cnt;
+    }
+    /* Check if message type is in the list of messages
```

Comment:
Should have a spacing line before a comment.

Reply:
Addressed in https://github.com/hanno-arm/mbedtls/commit/cf469458caf74173c045e62b508a602016f02c9a.